src/fileurlreader.cpp

```diff
@@ -23,16 +23,15 @@ nonstd::optional<std::string> FileUrlReader::reload()
 	tags.clear();
 	alltags.clear();
 
-	std::fstream f;
-	f.open(filename, std::fstream::in);
-	if (!f.is_open()) {
-		const auto error_message = strerror(errno);
-		return strprintf::fmt(_("Error: Failed to open file \"%s\" (%s)"),
+	std::vector<std::string> lines;
+	std::string error_message;
+	if (!utils::read_text_file(filename, lines, error_message)) {
+		return strprintf::fmt(_("Error: Failed to read urls from file \"%s\" (%s)"),
```

Comment:
I think this should say "URLs", because "urls" is just a file name.

Reply:
Agreed, done in https://github.com/newsboat/newsboat/pull/1332/commits/c9ded7daddfccb85c9fc1c7ba0cb94f38be79638